diem-move/diem-framework/DPN/releases/artifacts/current/build/DiemCoreFramework/sources/CRSN.move

```diff
@@ -65,7 +65,8 @@ module DiemFramework::CRSN {
 
     const MAX_CRSN_SIZE: u64 = 256;
 
-    public(script) fun allow_crsns(account: &signer) {
+    public(script) fun allow_crsns(account: signer) {
+        let account = &account;
```

Comment:
Nit: could you just use references in this function instead of let-binding to the `&account` reference here?

Reply:
sure was just trying to minimize git blame, but I can do that